### PR TITLE
Baseline issue template

### PR DESCRIPTION
**Pull request by charlesbvll**
#### What does this implement/fix? Explain your changes.

Adds a new issue template for requesting a new baseline. The template also contains instructions to implement such a baseline.

#### Any other comments?

I first need to write the doc about first contribution before marking the PR as ready.

### CODE REVIEW

**Review by danieljanes — CHANGES_REQUESTED** [review]

**Comment by charlesbvll** [comment]
Should I leave the issue title as is? Or change it to something similar to the current branch names (i.e.  Flower Baseline: [new_baeline_name])?

File: .github/ISSUE_TEMPLATE/baseline_request.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+name: Baseline request
+description: Suggest a new baseline
+labels: ["new baseline, good first issue"]
+title: "Add Flower Baseline: [new_baseline_name]"
+
+body:
+  - type: input
+    attributes:
+      label: Paper
+      description: What paper would you like implemented as a baseline ?
```
Comment (danieljanes) [inline .github/ISSUE_TEMPLATE/baseline_request.yml:10]:
```suggestion
      description: What paper / experiment would you like to see implemented as a Flower Baseline?
```

File: .github/ISSUE_TEMPLATE/baseline_request.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      placeholder: "Authors, year, title"
```
Comment (danieljanes) [inline .github/ISSUE_TEMPLATE/baseline_request.yml:11]:
```suggestion
      placeholder: "Authors, year, title, experiment"
```

File: .github/ISSUE_TEMPLATE/baseline_request.yml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Link
+      description: Provide a link (ideally an `arxiv.org/abs/*` link) to the abstract of the paper.
+      placeholder: https://arxiv.org/abs/2007.14390
```
Comment (danieljanes) [inline .github/ISSUE_TEMPLATE/baseline_request.yml:18]: Link to the Flower paper here

**Review by danieljanes — CHANGES_REQUESTED** [review]
Looks good, just a few small comments

File: .github/ISSUE_TEMPLATE/baseline_request.yml (hunk 3 above)
Comment (charlesbvll) [reply .github/ISSUE_TEMPLATE/baseline_request.yml:18]: This is already the case, except if you meant on line 17.

Comment (danieljanes) [reply .github/ISSUE_TEMPLATE/baseline_request.yml:18]: Right, sorry!

**Review by danieljanes — APPROVED** [review]
Looks great!

**Pull request merged**